connector/avro/src/test/scala/org/apache/spark/sql/execution/benchmark/AvroWriteBenchmark.scala

```diff
@@ -42,8 +44,8 @@ object AvroWriteBenchmark extends DataSourceWriteBenchmark {
     withTempPath { dir =>
       withTempTable("t1") {
         val width = 1000
-        val values = 500000
-        val files = 20
+        val values = 100000
```

Comment:
Make it smaller as bzip2 took about 20 minutes to compress the original size of data